ppml/trusted-big-data-ml/scala/docker-occlum/run_attestation_server.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -e
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" >/dev/null 2>&1 && pwd )"
+export RATLS_DIR="${script_dir}/../../ra_tls"
+
+function gen_secret_json() {
+    # First generate cert/key by openssl
+    bash ./gen-cert.sh
+
+    # Then do base64 encode
+    cert=$(base64 -w 0 flask.crt)
+    key=$(base64 -w 0 flask.key)
+    image_key=$(base64 -w 0 image_key)
+
+    # Then generate secret json
+    jq -n --arg cert "$cert" --arg key "$key" --arg image_key "$image_key" \
+        '{"flask_cert": $cert, "flask_key": $key, "image_key": $image_key}' >  secret_config.json
+}
+
+function build_server_instance()
+{
+    occlum gen-image-key image_key
+    gen_secret_json
+    rm -rf occlum_server && occlum new occlum_server
+    pushd occlum_server
+
+    jq '.verify_mr_enclave = "off" |
+        .verify_mr_signer = "off" |
+        .verify_isv_prod_id = "off" |
+        .verify_isv_svn = "off" |
+        .verify_enclave_debuggable = "on" |
+        .sgx_mrs[0].debuggable = false ' ../ra_config_template.json > dynamic_config.json
+
+    new_json="$(jq '.resource_limits.user_space_size = "500MB" |
+                    .metadata.debuggable = false ' Occlum.json)" && \
+    echo "${new_json}" > Occlum.json
+
+    rm -rf image
+    copy_bom -f ../ra_server.yaml --root image --include-dir /opt/occlum/etc/template
+
+    occlum build
+
+    popd
+}
+
+build_server_instance
+unset HTTPS_PROXY
+if [[ -z "$PCCS_URL" ]]; then
+    echo "NO PCCS"
```

Comment:
echo "[ERROR] NO PCCS"

Reply:
If PCCS or other env are not set correctly, we should `exit 1 `

Reply:
If add ATTESTATION env, no pccs is not an error.

Reply:
Updated here https://github.com/intel-analytics/BigDL/pull/4507/files#diff-51c5dc18fa529e5ddcf003d1622b47c94d970ac938bed6bf2fd146f0ee039469R86